low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yml

```diff
@@ -0,0 +1,220 @@
+   username: sspluser
+   password: gAAAAABf_F2Xh3y0x2rPTRGL0ld4f_u1AwI5pRHOOEk49-lSNsq339Mn8m9bSBsGH-U8bCaPnTJPW-LR4yQHc_CpSbhU3AL5wQ==
+   primary_rabbitmq_host: localhost
+
+RABBITMQEGRESSPROCESSOR:
+   virtual_host: SSPL
+   queue_name: sensor-queue
+   exchange_name: sspl-out
+   routing_key: sensor-key
+   ack_queue_name: sensor-queue
+   ack_exchange_name: sspl-out
+   ack_routing_key: sensor-key
+   username: sspluser
+   password: gAAAAABf_F2Xh3y0x2rPTRGL0ld4f_u1AwI5pRHOOEk49-lSNsq339Mn8m9bSBsGH-U8bCaPnTJPW-LR4yQHc_CpSbhU3AL5wQ==
+   message_signature_username: sspl-ll
+   message_signature_token: ALOIUD986798df69a8koDISLKJ282983
+   message_signature_expires: 3600
+   iem_route_addr: 
+   iem_route_exchange_name: sspl-out
+   primary_rabbitmq_host: localhost
+   limit_consul_memory: 50000000
+
+LOGGINGPROCESSOR:
+   virtual_host: SSPL
+   queue_name: iem-queue
+   exchange_name: sspl-in
+   routing_key: iem-key
+   username: sspluser
+   password: gAAAAABf_F2Xh3y0x2rPTRGL0ld4f_u1AwI5pRHOOEk49-lSNsq339Mn8m9bSBsGH-U8bCaPnTJPW-LR4yQHc_CpSbhU3AL5wQ==
+   primary_rabbitmq_host: localhost
+
+RABBITMQCLUSTER:
+   cluster_nodes: localhost
+   erlang_cookie: QLDZYPYEYGHECTHYQXFJ
+
+LOGGINGMSGHANDLER:
+   iem_routing_enabled: false
+   iem_log_locally: true
+
+DISKMSGHANDLER:
+   dmreport_file: /tmp/sspl/drivemanager/drive_manager.json
+   always_log_iem: false
+   max_drivemanager_events: 14
+   max_drivemanager_event_interval: 10
+
+NODEDATAMSGHANDLER:
+   transmit_interval: 10
+   units: MB
+   disk_usage_threshold: 80
+   cpu_usage_threshold: 80
+   host_memory_usage_threshold: 80
+
+RAIDSENSOR:
+   monitor: true
+   threaded: true
+   RAID_status_file: /proc/mdstat
+
+IPMI:
+   user: admin
+   pass: admin
+
+BMC_INTERFACE:
+   default: system
+
+REALSTORSENSORS:
+   monitor: true
+   polling_frequency: 30
+
+REALSTORPSUSENSOR:
+   threaded: true
+
+REALSTORFANSENSOR:
+   threaded: true
+
+REALSTORCONTROLLERSENSOR:
+   threaded: true
+
+REALSTORDISKSENSOR:
+   threaded: true
+   polling_frequency_override: 0
+
+REALSTORSIDEPLANEEXPANDERSENSOR:
+   threaded: true
+
+NODEHWSENSOR:
+   monitor: true
+   threaded: true
+   polling_interval: 30
+
+REALSTORLOGICALVOLUMESENSOR:
+   threaded: true
+
+REALSTORENCLOSURESENSOR:
+   threaded: true
+
+IEMSENSOR:
+   threaded: true
+   log_file_path: /var/log/cortx/iem/iem_messages
+   timestamp_file_path: /var/cortx/sspl/data/iem/last_processed_msg_time
+
+SYSTEMDWATCHDOG:
+   monitor: true
+   threaded: true
+   monitored_services: 
+   smart_test_interval: 999999999
+   run_smart_on_start: 'False'
+
+NODEHWACTUATOR:
+   ipmi_client: ipmitool
+
+DATASTORE:
+   store_type: consul
+   consul_host: 127.0.0.1
+   consul_port: 8500
+
+SASPORTSENSOR:
+  monitor: true
+  threaded: true
+  probe: sysfs
+
+MEMFAULTSENSOR:
+   monitor: true
+   threaded: true
+   probe: sysfs
+
+CPUFAULTSENSOR:
+   monitor: true
+   threaded: true
+   probe: sysfs
+
+STORAGE_ENCLOSURE:
+   mgmt_interface: cliapi
```

Comment:
Please ensure this is added in provisioner sheet where all the global common config present.

Reply:
for this key provisinor commented that this is not supported by them. So I have kept it in sspl config file.

Reply:
mgmt_interface need not be in global config, but in sspl.conf as its the interface SSPL uses for monitoring.

In LR R2 provisioning sheet, provisioner have already marked red for this one, stating they cant provide this.

Reply:
Okay.